src/main/java/com/minecolonies/coremod/commands/ActionMenuState.java

```diff
@@ -107,7 +107,7 @@ public EntityPlayerMP getPlayerForArgument(@NotNull final String argumentName)
         {
             if (argumentName.equals(actionArgument.getName()))
             {
-                if (ActionArgumentType.PLAYER == actionArgument.getType())
+                if (ActionArgumentType.ONLINE_PLAYER == actionArgument.getType() || ActionArgumentType.PLAYER == actionArgument.getType())
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1066)